src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -154,6 +155,49 @@ class PolicyEvaluatorServiceSpec extends FlatSpec with Matchers with TestSupport
     }
   }
 
+  "hasPermission" should "return true if given action is granted through membership in another policy" in {
```

Comment:
@dvoet can you review this test please?  thanks

Reply:
LGTM